rest_framework/views.py

```diff
@@ -134,12 +134,14 @@ def http_method_not_allowed(self, request, *args, **kwargs):
         """
         raise exceptions.MethodNotAllowed(request.method)
 
-    def permission_denied(self, request):
+    def permission_denied(self, request, message=None):
         """
         If request is not permitted, determine what kind of exception to raise.
         """
         if not request.successful_authenticator:
             raise exceptions.NotAuthenticated()
+        if message is not None:
+            raise exceptions.PermissionDenied(message)
         raise exceptions.PermissionDenied()
```

Comment:
Can we just simplify this to `raise exceptions.PermissionDenied(message=message)` - That'll still work fine if message is `None`, right?